services/src/test/groovy/org/jfrog/artifactory/client/BaseRepositoryTests.groovy

```diff
@@ -84,6 +91,32 @@ abstract class BaseRepositoryTests extends ArtifactoryTestsBase {
                     .customProperties(customProperties)
                     .build()
         }
+        if (prepareFederatedRepo) {
+            List<FederatedMember> members = new ArrayList<FederatedMember>()
+            if(federationUrl!=null && StringUtils.isNoneBlank(federationUrl))
```

Comment:
Missing white-space after "if" - according to our coding convemsions.

Reply:
fixed